src/task_behavior_ros/introspection.py

```diff
@@ -178,7 +179,7 @@ def _force_node(self, msg):
         for i, node_id in enumerate(msg.id):
             if node_id in self.node_map.keys():
                 rospy.loginfo(
-                    "forcing "+str(node_id)+" to " + str(msg.status[i].status))
+                    "forcing " + str(node_id) + " to " + str(msg.status[i].status))
```

Comment:
generally this is more efficient:
rospy.loginfo("forcing %s to %s", node_id, msg.status[i].status)